apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -430,6 +483,10 @@ private Long parseJWTExpirationDate(String token) {
 
       return null;
     }
+
+    private long getMinIdleMillisBeforeConnectionDrop() {
```

Comment:
Is this being used? It's private but I don't see it being referenced. (Also missing docs if it is being used)

Reply:
Yes it's being used on line 97.

Reply:
Comment added.